src/lib/parameters/parameters.cpp

```diff
@@ -885,6 +885,35 @@ param_reset_excludes(const char *excludes[], int num_excludes)
 	_param_notify_changes();
 }
 
+void
+param_reset_specific(const char *resets[], int num_resets)
+{
+	param_t	param;
+
+	for (param = 0; handle_in_range(param); param++) {
+		const char *name = param_name(param);
+		bool reset = false;
+
+		for (int index = 0; index < num_resets; index++) {
+			int len = strlen(resets[index]);
+
+			if ((resets[index][len - 1] == '*'
+			     && strncmp(name, resets[index], len - 1) == 0)
+			    || strcmp(name, resets[index]) == 0) {
+
+				reset = true;
+				break;
+			}
+		}
+
+		if (reset) {
+			param_reset(param);
+		}
+	}
+
+	_param_notify_changes();
```

Comment:
Not needed here
```suggestion
```

Reply:
Ah because `param_reset()` does it already :bulb: That means it's also not necessary to do it at the end of `param_reset_excludes()` right? I removed it there too.